src/clr-angular/data/datagrid/render/render-organizer.spec.ts

```diff
@@ -31,13 +37,21 @@ export default function(): void {
             // Initial sizing to make sure clearWidths is included in the next one.
             this.organizer.resize();
             let step = 0;
-            this.organizer.noLayout.subscribe(on => expect(step++).toBe(on ? 0 : 7));
-            this.organizer.clearWidths.subscribe(() => expect(step++).toBe(1));
-            this.organizer.detectStrictWidths.subscribe(() => expect(step++).toBe(2));
-            this.organizer.tableMode.subscribe(on => expect(step++).toBe(on ? 3 : 5));
-            this.organizer.computeWidths.subscribe(() => expect(step++).toBe(4));
-            this.organizer.alignColumns.subscribe(() => expect(step++).toBe(6));
-            this.organizer.scrollbar.subscribe(() => expect(step++).toBe(8));
+            this.organizer.renderStep.subscribe(renderStep => {
```

Comment:
I believe this works because everything essentially becomes synchronous, but this test is written in a way that if an observable was async that it would pass even if it failed. Should we use `async` to help with this?

Reply:
I'm not sure but I think we should keep it synchronous. The render function we are testing absolutely depends on this exact order to get calculations correct. 

Would it make sense to instead test that the renderStep is synchronous?